### PR TITLE
Like #78

### PR DESCRIPTION
**Pull request by samuelpampolini**
Implemented "like" for strings

### CODE REVIEW

File: src/Dommel/DommelMapper.SqlExpression.cs [hunk 1]
```diff
@@ -89,11 +96,67 @@ protected virtual object VisitExpression(Expression expression)
...
+                        break;
+                }
+
+                return expression;
+            }
+
+            /// <summary>
+            /// Process a Contains expression for string.
+            /// </summary>
+            /// <param name="expression">Method Call Expression.</param>
```
Comment (henkmollema) [inline src/Dommel/DommelMapper.SqlExpression.cs]:
```suggestion
            /// <param name="expression">The method call expression.</param>
```

File: test/Dommel.IntegrationTests/LikeTests.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Dommel.IntegrationTests
```
Comment (henkmollema) [inline test/Dommel.IntegrationTests/LikeTests.cs]: Can you put this test into the `Dommel.Tests` project? You're not using a database here so it's just a regular unit test.

Comment (henkmollema) [reply test/Dommel.IntegrationTests/LikeTests.cs]: @samuelpampolini can you take a look at this? You can just move the file to the Dommel.Tests project.

**Comment by henkmollema** [comment]
No worries, let me know when everything is ready. Thanks!

Op vr 19 okt. 2018 19:27 schreef Samuel Pampolini <notifications@github.com

**Comment by samuelpampolini** [comment]
@henkmollema everything has been changed.

**Comment by henkmollema** [comment]
Thanks a lot @samuelpampolini!

**Pull request merged**